packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -315,35 +321,40 @@ Iterable<Method> buildTags(
           toDartName(identifierBuilder.toString(), uppercaseFirstCharacter: true),
         );
 
-        final queryParams = <String>[];
-        for (final parameter in parameters) {
-          if (parameter.$in != openapi.ParameterType.query) {
-            continue;
-          }
+        if (!hasUriParameters) {
+          code.writeln("const _path = '${pathEntry.key}';");
```

Comment:
This could already parse the Uri instead of first storing the path as a variable.

Reply:
What exactly do you mean?
Uri does not have a constant constructor.

Reply:
But do we need it? In my eyes this makes the output code longer and a bit more complex which is not needed.

Reply:
Ok. 
What about leaving the path variable and just calling `Uri.Parse(path)` in the assignment for the request?
In theory we don't need it to be an URI at all as the DynamiteClient will just call `toString` to concatenate it with the baseUrl.

So just duplicate parsing :thinking:

Reply:
So you just want to pass the full path as a String instead of a Uri? Sounds like a good plan to remove duplicate parsing